api/server/router/system/system_routes.go

```diff
@@ -1,3 +1,6 @@
+// FIXME(thaJeztah): remove once we are a module; the go:build directive prevents go from downgrading language version to go1.16:
+//go:build go1.22
```

Comment:
nit: Does the `json.RawMessage(nil)` need go1.22?

Reply:
No, but the `any` does require go .. whatever version added generics 😞 

I was on the fence on changing it to `interface{}`, which would be the alternative

Reply:
Ohh right, I missed the `any` 😄